cmd/controller-manager/main.go

```diff
@@ -211,7 +211,9 @@ func main() {
 	}
 
 	// Add the Metrics Service
-	addMetrics(ctx, cfg, namespace)
+	if monitoringEnabled == "true" {
```

Comment:
Suggest we change above to use:

	monitoringEnabled := util.GetBooleanEnv("ENABLE_MONITORING")